src/main/java/com/maplexpbar/MapleXPBarPlugin.java

```diff
@@ -7,8 +7,11 @@
 import lombok.AllArgsConstructor;
 import lombok.Getter;
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.*;
```

Comment:
I think that it's unnecessary to use the whole library instead of just importing the three classes.